px4_ros2_cpp/include/px4_ros2/navigation/experimental/global_navigation_interface.hpp

```diff
@@ -0,0 +1,76 @@
+/****************************************************************************
+ * Copyright (c) 2023 PX4 Development Team.
+ * SPDX-License-Identifier: BSD-3-Clause
+ ****************************************************************************/
+
+#pragma once
+
+#include <optional>
+#include <rclcpp/rclcpp.hpp>
+#include <Eigen/Eigen>
+
+#include <px4_msgs/msg/vehicle_global_position.hpp>
+#include <px4_ros2/navigation/experimental/navigation_interface_base.hpp>
+
+using namespace Eigen;
+using namespace px4_msgs::msg;
+
+namespace px4_ros2
+{
+
+struct GlobalPositionEstimate
+{
+  rclcpp::Time timestamp_sample {};
+
+  // Lat lon
+  std::optional<Vector2d> lat_lon {std::nullopt};
+  // Standard deviation of horizontal position error (metres)
+  std::optional<float> lat_lon_stddev {std::nullopt};
```

Comment:
@DanMesh Is this the name we want to expose `lat_lon_stddev`? Or should we use `eph` instead. The former is more descriptive, but the latter is closer to the VehicleGlobalPosition message.

Reply:
@bresch any opinion on this?

Reply:
I think eph and epv are widely used

Reply:
What about using a variance instead of a standard deviation? e.g. `lat_lon_variance` and `altitude_variance`.

This would then be consistent with the local navigation interface, which uses variances.

Reply:
discussion decision: use `horizontal_variance` and `vertical_variance`